Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/utils/StartupHandler.java

```diff
@@ -71,13 +70,13 @@ public static void setSessionTime(Context context)
    */
   public static void checkSessionReset(Context context)
   {
-    long currentTime = new Date(System.currentTimeMillis()).getTime();
     SharedPreferences preferences = PreferenceManager.getDefaultSharedPreferences(context);
     long lastOpen = preferences.getLong(LAST_CLOSED, 0);
-    if (currentTime > (lastOpen + SESSION_TIMEOUT))
+    final Instant current = Instant.now();
+    final Instant lastOpened = Instant.ofEpochMilli(lastOpen).plus(6, ChronoUnit.HOURS);
+    if (current.isAfter(lastOpened))
```

Comment:
I think it would make more sense like this:

```suggestion
    final Instant lastOpened = Instant.ofEpochMilli(lastOpen);
    if (current.isAfter(lastOpened.plus(6, ChronoUnit.HOURS)))
```

Otherwise `lastOpened` is technically misnamed.

Reply:
Thanks for the suggestion.